requirements.txt

```diff
@@ -10,3 +10,4 @@ shapely
 boto3
 botocore
 pyproj
+aenum
```

Comment:
Coincidentally I found out that if `aenum<=2.1.2` function `extend_enums` doesn't work correctly (at least for some versions). Therefore I recommend setting here a condition `aenum>=2.1.4`.